Akka-apps and akka-fsesl fail to start due starting before redis is running fully

<!--PLEASE DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
This issue tracker is only for bbb development related issues.-->

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. `systemctl restart bigbluebutton.target`
2. `bbb-conf --status`

**Expected behavior**
```
nginx —————————————————► [✔ - active]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✔ - active]
bbb-fsesl-akka ————————► [✔ - active]
mongod ————————————————► [✔ - active]
bbb-html5 —————————————► [✔ - active]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
bbb-html5-backend@1 ———► [✔ - active]
bbb-html5-backend@2 ———► [✔ - active]
bbb-html5-frontend@1 ——► [✔ - active]
bbb-html5-frontend@2 ——► [✔ - active]
etherpad ——————————————► [✔ - active]
bbb-web ———————————————► [✔ - active]
bbb-pads ——————————————► [✔ - active]
```

**Actual behavior**
```
nginx —————————————————► [✔ - active]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✘ - failed]
bbb-fsesl-akka ————————► [✘ - failed]
mongod ————————————————► [✔ - active]
bbb-html5 —————————————► [✔ - active]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
bbb-html5-backend@1 ———► [✔ - active]
bbb-html5-backend@2 ———► [✔ - active]
bbb-html5-frontend@1 ——► [✔ - active]
bbb-html5-frontend@2 ——► [✔ - active]
etherpad ——————————————► [✔ - active]
bbb-web ———————————————► [✔ - active]
bbb-pads ——————————————► [✔ - active]
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**BBB version:**
BigBlueButton 2.6.0-alpha.2 with bigbluebutton.target PRs merged #14616 #15334

@danimo is assisting (and has been assisting :rocket: ) Thank you!!

**Additional context**
Logs

```
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]: Exception in thread "main" io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.lettuce.core.RedisConnectionException.create(RedisConnectionException.java:78)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.lettuce.core.RedisConnectionException.create(RedisConnectionException.java:56)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.lettuce.core.AbstractRedisClient.getConnection(AbstractRedisClient.java:330)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.lettuce.core.RedisClient.connectPubSub(RedisClient.java:336)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at org.bigbluebutton.common2.redis.RedisPublisher.<init>(RedisPublisher.scala:21)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at org.bigbluebutton.Boot$.delayedEndpoint$org$bigbluebutton$Boot$1(Boot.scala:40)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at org.bigbluebutton.Boot$delayedInit$body.apply(Boot.scala:19)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.Function0.apply$mcV$sp(Function0.scala:39)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.Function0.apply$mcV$sp$(Function0.scala:39)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.App.$anonfun$main$1(App.scala:76)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.App.$anonfun$main$1$adapted(App.scala:76)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.collection.AbstractIterable.foreach(Iterable.scala:919)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.App.main(App.scala:76)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at scala.App.main$(App.scala:74)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at org.bigbluebutton.Boot$.main(Boot.scala:19)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at org.bigbluebutton.Boot.main(Boot.scala)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]: Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: /127.0.0.1:6379
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]: Caused by: java.net.ConnectException: Connection refused
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
Jul 07 14:06:03 anton-temp-26-target bbb-apps-akka[42881]:         at java.base/java.lang.Thread.run(Thread.java:829)
Jul 07 14:15:49 anton-temp-26-target systemd[1]: Stopping BigBlueButton Apps (Akka)...
Jul 07 14:15:50 anton-temp-26-target systemd[1]: bbb-apps-akka.service: Main process exited, code=exited, status=143/n/a
Jul 07 14:15:50 anton-temp-26-target systemd[1]: bbb-apps-akka.service: Failed with result 'exit-code'.
Jul 07 14:15:50 anton-temp-26-target systemd[1]: Stopped BigBlueButton Apps (Akka).
```

--- a/build/packages-template/bbb-config/before-install.sh
+++ b/build/packages-template/bbb-config/before-install.sh
@@ -1,0 +1,20 @@
+#!/bin/bash -e
+
+case "$1" in
+    install|upgrade|1|2)
+
+        # We want to ensure Redis is fully started before it signals it is ready to rely on
+        mkdir -p /etc/systemd/system/redis-server.service.d
+        cat <<HERE > /etc/systemd/system/redis-server.service.d/overrride.conf
+[Service]
+ExecStartPost=/usr/bin/timeout 30 sh -c 'while ! ss -H -t -l -n sport = :6379 | grep -q "^LISTEN.*:6379"; do sleep 1; done'
+HERE
+    ;;
+
+    abort-upgrade)
+    ;;
+
+    *)
+        echo "## preinst called with unknown argument \`$1'" >&2
+    ;;
+esac

--- a/build/packages-template/bbb-config/build.sh
+++ b/build/packages-template/bbb-config/build.sh
@@ -60,6 +60,7 @@ cp bigbluebutton.target staging/lib/systemd/system/
 fpm -s dir -C ./staging -n $PACKAGE \
     --version $VERSION --epoch $EPOCH \
     --after-install after-install.sh \
+    --before-install before-install.sh \
     --description "BigBlueButton configuration utilities" \
     $DIRECTORIES \
     $OPTS \

